doc/config.rst

```diff
@@ -0,0 +1,670 @@
+
+If you do not turn these on, you should use the parameterized optics available in the atmospheric PSF instead.  You have three choices: they are set with the the
+
+Photon Operation Type
+---------------------
+
+type: **RubinOptics**
+^^^^^^^^^^^^^^^^^^^^^
+Photons ray-traced though the Rubin optical system.
+
+Required keywords to set:
+""""""""""""""""""""""""""
+
+    * ``boresight`` = *RaDec_value* (default = None) The CelestialCoord of the boresight of the observation.
+    * ``camera`` = *str_value* (default = None) The name of the camera to use.
+
+type: **RubinDiffractionOptics**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Ray-traced photons including the effects of diffraction when passing through edges like the telescope spiders.
+
+Required keywords to set:
+"""""""""""""""""""""""""
+
+    * ``boresight`` = *RaDec_value* (default = None) The CelestialCoord of the boresight of the observation.
+    * ``camera`` = *str_value* (default = None) The name of the camera to use.
+    * ``azimuth`` = *float_value* (default = None) The azimuth angle in degrees.
+    * ``altitude`` = *float_value* (default = None) The altitude angle in degrees.
+
+
+Optional keywords to set:
+""""""""""""""""""""""""""
+
+    * ``latitude`` = *bool_value* (default = True) The latitude of the observatory.
+    * ``disable_field_rotation`` = *bool_value* (default = False) Do not consider the effect of the rotation of the camera relative to the spiders of the telescope during the exposure.
+
+type: **RubinDiffraction**
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Diffractive effects only during the FFT.
+
+Required keywords to set:
+"""""""""""""""""""""""""
+
+    * ``azimuth`` = *float_value* (default = None) The azimuth angle in degrees.
+    * ``altitude`` = *float_value* (default = None) The altitude angle in degrees.
+    * ``latitude`` = *bool_value* (default = True) The latitude of the observatory.
+
+Optional keywords to set:
+""""""""""""""""""""""""""
+
+    * ``disable_field_rotation`` = *bool_value* (default = False) Do not consider the effect of the rotation of the camera relative to the spiders of the telescope during the exposure.
+
+Output types
+============
+
+The output field is used to specify where to write output files and what format they should be.  There are several possibilities, including FITS files before and after electronics readout, and various types of truth information.  *imSim* adds the ``LSST_CCD`` type. It  understands how to write "eimage" files which are true representations of the electrons in the CCD including signals from the objects and cosmic rays with important physics effects such as the brighter-fatter effect and tree-rings applied.
```

Comment:
This isn't really correct.  The tree rings and brighter fatter are both done by LSST_Image, not LSST_CCD.  I mean, yes, those are in the eimage, but only because we specify them in the stamp section, not anything to do with the output section.

LSST_CCD mostly does the following:
1. Set the det_num as something that can be used in other places in eval statements or base['det_num']
2. Likewise det_name, which is derived from that.
3. Set up the xsize and ysize parameters based on the camera specification for this det_num.
4. Add cosmic rays (optionally)
5. Write a bunch of things to the image header as needed for DM processing.

Reply:
Clarify text.